setup.py

```diff
@@ -37,7 +37,7 @@
     include_package_data=True,
     install_requires=[
         'celery',
-        'django',
+        'django==3.2a1',
```

Comment:
Due to this bug: https://code.djangoproject.com/ticket/31910

Reply:
It'd be good to note this as a code comment. That should be more persistent.

Reply:
Will do